src/__tests__/styling-utils.spec.js

```diff
@@ -0,0 +1,96 @@
+import {
+  getAxisLabelStyle,
+  getLegendLabelStyle,
+  getLegendTitleStyle,
+  getValueLabelStyle,
+  labelStylingDefinition,
+} from '../styling-utils';
+
+describe('Label styling definition', () => {
+  const path = 'example.path';
+  const chartId = 'mekkochart';
+  const mockFontResolver = {
+    getOptions: () => {
+      ['font1', 'font2', 'font3'];
+    },
+    getDefaultValue: () => {
+      'defaultValue';
+    },
+  };
+  const mockTheme = {
+    getStyle: () => {
+      'green';
+    },
+  };
+  const definition = labelStylingDefinition(path, mockFontResolver, chartId, mockTheme);
+
+  expect(definition.fontFamilyItem.ref).equal('example.path.fontFamily');
```

Comment:
Have not seen expect calls outside of the test function before. How does this work when the expect fails?